src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -3454,8 +3454,8 @@ private void recordRelocation(void* location, void* target, ushort fRelocType, u
             Debug.Assert(locationBlock != BlockType.Unknown, "BlockType.Unknown not expected");
 
             // TODO: Arbitrary relocs
-            if (locationBlock != BlockType.Code)
-                throw new NotImplementedException("Arbitrary relocs");
+            // if (locationBlock != BlockType.Code)
```

Comment:
I do not think we want to just comment this out.

If you need to disable this for ARM to make progress, it should be disabled just for ARM only. Ideally, there should be link to an issue that tracks implementing it.